src/ultimate_notion/obj_api/core.py

```diff
@@ -191,16 +192,22 @@ def _resolve_type(cls, value: Any, handler: ValidatorFunctionWrapHandler):
         type_name = value.get('type')
 
         if type_name is None:
-            msg = "Missing 'type' in data"
-            raise ValueError(msg)
+            logger.debug(f'Missing type in data {value} - breaking recursion')
+            return handler(value)
```

Comment:
@al1p Not sure I understand why these two lines changed. Was this necessary for it to work?

Reply:
It's a hack but I encountered a case of infinite loop when the type was None. This allowed me to break the recursion and move on... It worked out for me but I'm not sure of the side effects.